docs/release-notes/release-notes-0.14.0.md

```diff
@@ -7,4 +7,8 @@ added](https://github.com/lightningnetwork/lnd/pull/5520) to ensure that all
 PRs add an entry in the release notes folder that at leasts links to PR being
 added.
 
+## Code cleanup, refactor, typo fixes
+Removed unused error check (https://github.com/lightningnetwork/lnd/pull/5537).
```

Comment:
Since there is no main `# Code health` section, can you add that before line 10 please? Also I think we should use bullet points here as there are going to be a bunch of these and a whole paragraph each is a bit too much IMO.

Reply:
Np.

Reply:
But you still want a short description right...not just a link or pull request id?

Reply:
Yes, a short description is fine, thanks.